pnet/Private-Networks-PSK-V1.md

```diff
@@ -0,0 +1,75 @@
+## Pre-shared Key Based Private Networks in IPFS
+
+This document describes the first version of private networks (PN) featured in IPFS.
```

Comment:
We should settle on Pre-Shared Key (PSK) or PN for the name, so that there is no ambiguity (I would prefer to call it PSK networks as people know what that means directly from using wifi). We will have other kinds of Private Networks.